webTemplate/qbgrow.com/magen/iot-admin/combinedCode/combined.py

```diff
@@ -1,55 +1,60 @@
-# from ds18b20 import loop, sensor, read, kill
+from twilio.rest import Client
+from ds18b20 import loop, sensor, read, kill
 import sqlite3 as sql
-
-# from gy21 import humidity
-# import PCF8591 as ADC
-# from moisture import measure
-# from temp2 import read
-import random
+from gy21 import humidity
+import PCF8591 as ADC
+from moisture import measure
+from temp2 import read
 import time
-
 connection = sql.connect("IoTDatabase.db")
 cur = connection.cursor()
-import utils
 
-# serialNum = sensor()
-# print("Soil tempersature",loop(serialNum))
-# print("The humidity",humidity())
-# print("Soil moisture", measure())
-# sleep(0.2)
-# print("Atmospheric temp", read())
+#Twilio Account Details
+
+account_sid = 'AC1c805cf3f05a5ab71eb7c21e35b7af33'
```

Comment:
Its very bad to publish such details on the internet. Especially with open source projects as this.